### ISSUE TITLE
Mouseover cursor isnt the same as non mouseover anymore

### ISSUE THREAD

**Issue by makedir** [PandaTeemo/YoloMouse#3]
Please give option to change this, it is really annoying to have a different cursor when mouse over, really bad design decision. I just want to change the color not the complete cursor, especially not into a circle one.

**Comment by PandaTeemo** [comment]
youd need to take the original game mouse cursor and modify it yourself and
add it as a custom cursor to YM. its too tricky to automatically change
existing game cursors to some new color.

otherwise im not entirely sure what youre asking :)

**Comment by makedir** [comment]
No... you did not understand. Since I updated to 0.6.1beta you changed the mouseover cursor to always being the round annoying way too big circle. Before 0.6.1 the default behavior was pink/blue: http://i.imgur.com/jqNOQmK.png => http://i.imgur.com/g9MUxBq.png or vice versa, switching the color when mouseover. Now it switches from: http://i.imgur.com/g9MUxBq.png to http://i.imgur.com/ODGdYmI.png . Please give an option to change this behavior back.

**Comment by PandaTeemo** [comment]
oh that, the new behavior is hitting "ctrl alt shift 1" repeatedly cycles
through 10 different arrow cursors, "ctrl alt shift 2" cycles through the
circles, etc. this was a necessary change to handle way more cursors.

**Comment by makedir** [comment]
What? No... O_0 you still didnt get it. IT IS ALWAYS A CIRCLE WHEN MOUSEOVER, this is terrible. Please change it back or give option to change it. It wasnt before 0.6.1. that way. I want the cursor to change colors not change always into a circle.

**Comment by PandaTeemo** [comment]
yes, just use CTRL ALT SHIFT 1 multiple times for all different arrow
cursors.... 2 is for circles 3 is for cross

you may have to change existing cursors when upgrading, including mouseover.

it has to be this way to handle many cursors, you can still download old
version.

**Comment by makedir** [comment]
DUDE. Are you serious?? How many times shall I tell you, how many times to do you need to understand this problem?

CTRL ALT SHIFT 1 CHANGES THE NON-MOUSEOVER http://i.imgur.com/jqNOQmK.png CURSOR
CTRL ALT SHIFT 2 CHANGES THE NON-MOUSEOVER circle cursor

I AM USING CURSOR 1 FOR NON-MOUSEOVER: CTRL ALT SHIFT 1

PROBLEM/BUG: IT IS ALWAYS CURSOR 2 FOR MOUSEOVER, THIS IS A BUG AND NOT WANTED BEHAVIOR.

Dont you understand, do I need to record a video with my phone dammit??

Problem 1: regardless of the CTRL ALT SHIFT 1-0, the mouseover IS ALWAYS THE PINK CIRCLE
Problem 2: pink mouseover cursor IS ALWAYS THE SAME SIZE

**Comment by makedir** [comment]
Respond please? Did you understand now the two issues?

**Comment by PandaTeemo** [comment]
here was reply....

----
theres no concept of mouseover in YM. its just whatever cursor you see you
change with hotkey.

maybe you should specify the game youre talking about. some games, like d3,
change the mouseover cursor to something else while you hold down SHIFT or
ALT preventing YM from changing the intended cursor.

**Comment by makedir** [comment]
https://dl.dropboxusercontent.com/u/14937594/yolo.mp4

**Comment by PandaTeemo** [comment]
if you cant reassign that mouseover cursor (using YM hotkeys) i think itd
easiest for you to just delete the ...diablo3... file in

C:\Users\...\AppData\Local\YoloMouse\

then restart yolomouse and d3. this will clear all assigned cursors and let
you reassign everything.

**Comment by makedir** [comment]
Could you for a moment please stop being so ignorant, I know it is hard as a developer and maker of a program, I tend to do it myself and listen to me maybe for one second.

THIS IS A DAMN BUG, no TWO:

Why dont you understand this problem? Look at the damn video.

1. regardless of the assigned MAIN cursor, whenever you mouseover over a monster in Diablo 3, YoloMouse changes the cursor to a CIRCLE, ALWAYS A CIRCLE.

2. the ALWAYS CIRCLE IS ALWAYS THE SAME SIZE, hotkey size is IGNORED FOR THE MOUSEOVER CURSOR.

**Comment by PandaTeemo** [comment]
if hotkeys dont change mouseover cursor then its a bug. but its working for
everyone else so its probably specific to your setup. clearing old settings
and redoing it should clear it as stated in last email.

**Comment by makedir** [comment]
... How do I deactivate all hotkeys, it seems there is no option for this in the ini file. There need to be some setting like allhotkeysoff.

**Comment by PandaTeemo** [comment]
theres no point to this, just delete the diablo3 ini file and restart d3/ym

**Comment by makedir** [comment]
I am not asking without a reason. Could you just answer to my damn questions, yes. If I ask something there is a reason for. Please give an option to disable any hotkeys or give a hotkey to toggle them on/off.

**Comment by PandaTeemo** [comment]
you can just remove hotkey bindings in settings.ini effectively disabling
them

**Comment by makedir** [comment]
So if I change them to

CursorKey2=

it will already work / disable them?

**Comment by PandaTeemo** [comment]
yes, but theres little reason to do this

**Comment by makedir** [comment]
Yes there is big reason to do that. I know from a developers point of view there is "always no reason" or "always a reason" to do something or to not give any options/settings at all. I am a developer myself and I know how hard it is to step away from something you created and look at it from a users point of view, or from a objective point of view. I tend to do it myself, sadly most stand-alone developers love their "creation" too much to acknowledge improvements or just simple toggle options to turn stuff on and even more important, off. Developers tend to change something they think is a total improvement whre it is not and worse, never give any simple options to turn those changes off or back.

**Comment by PandaTeemo** [comment]
well im just saying disabling hotkeys doesnt really change anything, but
the option does exist by editing settings.ini. i may make a more friendly
UI for this later though.

hopefully wiping the d3 ini fixed your problem of the stuck circle binding.
